objmodel/chapter.md

```diff
@@ -202,26 +202,26 @@ class Class(Base):
         self.base_class = base_class
 ````
 
-Now we can also see what a metaclass is. Since classes are also a kind of
-object, they (indirectly) inherit from ``Base``. Thus the class needs to be an instance of
-another class, its metaclass.
+Since classes are also a kind of
+object, they (indirectly) inherit from ``Base``. Thus, the class needs to be an instance of
+another class: its metaclass.
 
-Now our first test above almost passes. The only missing bit is the definition
+Now our first test almost passes. The only missing bit is the definition
 of the base classes ``TYPE`` and ``OBJECT``, which are both instances of
 ``Class``. For these we will make a major departure from the Smalltalk model,
-which has a fairly complex metaclass system. Instead we will use the model of ObjVlisp, which is
-also the one that Python adopted (footnote: P. Cointe, “Metaclasses are first
+which has a fairly complex metaclass system. Instead we will use the model used by ObjVlisp, which 
```

Comment:
Maybe "Instead we will use the model introduced by ObjVlisp"? I dislike the double "use".